tidb-cloud/data-service-postman-integration.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Run Data App in Postman
+summary: Learn how to run your Data App in Postman.
+---
+
+# Run Data App in Postman
+
+[Postman](https://www.postman.com/) is an API platform for building and using APIs, which simplifies each step of the API lifecycle and streamlines collaboration so you can create better APIs—faster.
+
+In TiDB Cloud [Data Service](https://tidbcloud.com/console/data-service), you can easily import your Data App to Postman and leverage Postman's extensive tools to enhance your API development experience.
+
+This document describes how to import your Data App to Postman and how to run your Data App in Postman.
+
+## Before you begin
+
+Before importing a Data App to Postman, make sure that you have the following:
+
+- A [Postman](https://www.postman.com/) account.
+- A [Postman desktop app](https://www.postman.com/downloads) (Optional. You can use the Postman webpage version without downloading the app.)
+- A [Data App](/tidb-cloud/data-service-manage-data-app.md) with at least one [endpoint](/tidb-cloud/data-service-manage-endpoint.md). Only endpoints that meet the following requirements can be imported to Postman:
+
+    - The target cluster is selected.
+    - The endpoint path and request method are configured.
+    - The SQL statements are written.
+
+- An [API key](/tidb-cloud/data-service-api-key.md#create-an-api-key) for the Data App.
+
+## Step 1. Import your Data App to Postman
+
+To import your Data App to Postman, take the following steps:
+
+1. In the [TiDB Cloud console](https://tidbcloud.com), navigate to the [**Data Service**](https://tidbcloud.com/console/data-service) page of your project.
+2. In the left pane, click the name of your target Data App to view its details.
+3. In the upper-right corner of the page, click **Run in Postman**. A dialog for import instructions is displayed.
```

Comment:
A dialog with import instructions will be displayed.

Reply:
In user docs, we usually use present tense rather than future tense.

https://developers.google.cn/style/tense
![image](https://github.com/pingcap/docs/assets/79440533/0e9bcf19-496a-4ff5-9028-59be3bdb95cd)